infection.json

```diff
@@ -0,0 +1,19 @@
+{
+    "$schema": "vendor/infection/infection/resources/schema.json",
+    "source": {
+        "directories": [
+            "src"
+        ]
+    },
+    "logs": {
+        "text": ".infection/infection.log",
```

Comment:
Suggestion: `"github": true` too - that will give you some decent github output in future :D